### ISSUE TITLE
Canvas not aligned with base map in React

### ISSUE THREAD

**Issue by Darune** [uber/deck.gl#4115]
Using deck.gl v8 binary attribute support
I try to put a simple point on Paris using the following react component declaration / props
```
<DeckGL
    layers={[
        new PointCloudLayer({
            id: 'test-paris',
            data: {
                length: 1,
                attributes: {
                    getPosition: {value: new Float32Array([2.3522, 48.8566, 0.0]), size: 3}
                }
            },
            filled: true,
            getRadiusPixel: 20,
            getColor: d => [255, 140, 0],
        })
    ]}
    initialViewState={{
        latitude: 48.8566,
        longitude: 2.3522,
        zoom: 6,
    }}
    controller={true}
>
    <ReactMapGL
        key="map"
        mapStyle="mapbox://styles/mapbox/dark-v10"
        mapboxApiAccessToken="<token>"
    />
</DeckGL>
```
But the Point is displayed on the right side of the screen no matter the zoom level. 
Is there anything I'm missing regarding using the binary attributes with React ?

**Comment by Pessimistress** [comment]
What deck.gl version are you using?

**Comment by Darune** [comment]
I was on 8.0.3
same issue on 8.0.4
here is a sandbox:
https://codesandbox.io/s/wonderful-waterfall-x33o4?fontsize=14&hidenavigation=1&theme=dark

in case this is client related, here is a short video of what i'm seeing 

![deck gl-binary-attributes](https://user-images.githubusercontent.com/783455/72045878-774a8d80-32b7-11ea-8155-e7a30a574f2e.gif)

**Comment by Pessimistress** [comment]
Ah the bug is actually not in binary data. If you remove the css `text-align: center;` it will be fixed. This is a scenario we failed to account for. We should explicitly set the CSS on the React component.

Just FYI, the default export from react-map-gl (i.e. `InteractiveMap`) is not effective as a child of `<DeckGL>` because the deck canvas captures all events. It's more efficient to use

https://github.com/uber/deck.gl/blob/86e454e521e33d4bf821cd22399f07f5a5308c8d/examples/website/scatterplot/app.js#L3

https://github.com/uber/deck.gl/blob/86e454e521e33d4bf821cd22399f07f5a5308c8d/examples/website/scatterplot/app.js#L54-L61

**Comment by Darune** [comment]
Thank you, the issue was indeed the `text-align` css.

Missed the StaticMap paragraph in the documentation, thank you for pointing that out.

**Comment by Pessimistress** [comment]
CSS position fixed in 8.0.5.
